doc/Apptools_Test_Suite_User_Guide.md

```diff
@@ -0,0 +1,185 @@
+# Apptools Test Suite User Guide
+
+## Introduction
+
+This document provides method to run Apptools TestSuite. Currently the target platforms are Android, iOS, Linux and Windows
+
+## Setup Environment
+
+### Linux host
```

Comment:
The first letter of each word in title should be capital, except conj such as and, of ...